frameworks/Go/go-raw/src/hello/hello.go

```diff
@@ -187,6 +225,30 @@ func queriesHandler(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(world)
 }
 
+func queriesPreparedHandler(w http.ResponseWriter, r *http.Request) {
+	n := 1
+	if nStr := r.URL.Query().Get("queries"); len(nStr) > 0 {
+		n, _ = strconv.Atoi(nStr)
+	}
+
+	if n < 1 {
+		n = 1
+	} else if n > 500 {
+		n = 500
+	}
+
```

Comment:
This code (from line 229) is duplicated 4 times now. May be extract it in a separate method? (`getQueriesParam(r *http.Request)`)